### ISSUE TITLE
GoogleAnalyticsClientError

### ISSUE THREAD

**Issue by shahanar** [clintecker/python-googleanalytics#13]
Hi,

  when i try to connect with following code it raises "GoogleAnalyticsClientError"  GAError: HTTP Error 404: Not Found 

code sample

connection = Connection('example@gmail.com', 'password')
account = connection.get_account('12345678')

**Comment by nitishk** [comment]
i am getting the same error please look at it

**Comment by nitishk** [comment]
I am able to connect but when i try to get account it showing the error as mentioned above. I did some research and i think the error lies in path for getting account '/analytics/feeds/accounts/default/'.

**Comment by defrex** [comment]
I'm getting the same error. It only started recently.

**Comment by defrex** [comment]
Looks like they shut down the 2.3 version of the API, which removed the Account endpoint.

https://developers.google.com/analytics/resources/articles/gdata-migration-guide

**Comment by nitishk** [comment]
@defrex
Yes, i studied the link you have posted here and surely account feeds are no longer supported so we need to make calls through management API. 
did u try using google analytics python client library?
